TASK: Disable normal Dependabot updates

Sebobo opened the pull request:
They don't really have any value for us as we run updates manually when needed. Security updates are still enabled separately.

Sebobo committed:
TASK: Disable normal Dependabot updates

They don't really have any value for us as we run updates manually when needed. Security updates are still enabled separately.

Sebobo commented:
I'm not sure what the target branch for composer means exactly 🤷🏻‍♂️

mhsdesign reviewed (approved):
👍

markusguenther commented on .github/dependabot.yml:
```diff
@@ -1,20 +1,11 @@
-    directory: "/"
-    schedule:
-      interval: "daily"
-    target-branch: "5.3"
```
We can change that to 7.3 as well

markusguenther commented on .github/dependabot.yml:
```diff
@@ -1,20 +1,11 @@
 version: 2
 updates:
 
-  # Maintain dependencies for npm
```
We can keep the block and just add `open-pull-requests-limit: 0`
That will lead to security updated only and disable regular version updates.

See: https://docs.github.com/en/enterprise-server@3.3/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates

markusguenther committed:
TASK: Configure security updates only for NPM

For more information visit https://docs.github.com/en/enterprise-server@3.3/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates
